recipe/meta.yaml

```diff
@@ -19,14 +19,13 @@ requirements:
   host:
     - python
     - pip
-    - jupyter-packaging >=0.9,<2
-    - jupyter_server
     - setuptools
     - wheel
+    - hatchling >=0.25
   run:
     - python
     - babel
-    - entrypoints >=0.2.2
+    - importlib-metadata >=3.6
```

Comment:
Should be `importlib-metadata >=3.6  # [py<310]`, see https://github.com/jupyterlab/jupyterlab_server/blob/ca2fc08221673126b30e73750ad9fc213daa5c50/pyproject.toml#L27

Reply:
The adjustments were made for `importlib-metadata`:
https://github.com/AnacondaRecipes/jupyterlab_server-feedstock/blob/66ee2952c55b2a2974574f84878338d24dc3b63f/recipe/meta.yaml#L26